src/sharedData/components/SharedDataEntryBase.js

```diff
@@ -215,8 +215,10 @@ const SharedDataEntryBase = props => {
         </Grid>
         <Grid item xs={1} order={{ xs: 7 }} display={{ md: 'none' }} />
         <Grid item xs={11} md={3} order={{ xs: 8, md: 4 }}>
-          {formatDate(i18n.resolvedLanguage, dataEntry.createdAt)}, by{' '}
-          {t('user.full_name', { user: dataEntry.createdBy })}
+          {t('sharedData.file_date_and_author', {
+            date: formatDate(i18n.resolvedLanguage, dataEntry.createdAt),
+            user: dataEntry.createdBy,
```

Comment:
```suggestion
            user: dataEntry.createdBy.full_name,
```

Reply:
This doesn't work. The `dataEntry.createdBy` object only has firstName and lastName (and id) members. There are other strings in the strings file that use `$t(user.full_name)` -- are we trying to avoid that? If so, this line could be:
`name: t('user.full_name', { user: dataEntry.createdBy })` -- is that preferred?

Reply:
@josebui Do you have an opinion on the preferred way to do this? Is it weird to have a t() inside another t()?

Reply:
`$t(user.full_name)` seems to be a common way of doing this, I mean inside the localized files, since the name structure is linked to specific locales I think it makes more sese to decide wheter we want to use full name or any other variation of name for each locale. To me that means that ideally from the code you pass the user and then in the localized file you decide how to do this for each locale